opensplice_cmake_module/config/ros_ospl.xml

```diff
@@ -21,5 +21,21 @@
             <!-- the following one is necessary only for TwinOaks CoreDX DDS compatibility -->
             <!-- <ExplicitlyPublishQosSetToDefault>true</ExplicitlyPublishQosSetToDefault> -->
         </Compatibility>
+        <Discovery>
+             <!-- By default, OpenSplice only allows up to 9 participants (see
+                  http://download.prismtech.com/docs/Vortex/html/ospl/ConfGuide/guide.html#maxautoparticipantindex)
+                  This means that only 9 nodes can be run at a time, which even
+                  moderately complex robots easily achieve.  From code inspection
+                  on OpenSplice 6.9 20181126, there are two minor downsides to increasing this:
+
+                  1.  Slightly slower startup time since the code iterates over every
+                  participant index up to the max to find a free one
+                  (see https://github.com/ADLINK-IST/opensplice/blob/OSPL_V6_9_181126OSS_RELEASE/src/services/ddsi2e/core/q_init.c#L890)
+                  2.  Slightly slower port choosing when non-multicast addresses are in
+                  use, again because of iterating over every participant index
+                  (see https://github.com/ADLINK-IST/opensplice/blob/OSPL_V6_9_181126OSS_RELEASE/src/services/ddsi2e/core/q_addrset.c#L93)
```

Comment:
Please move this additional information to the ticket description rather than the xml file.